modules/ztp-pulling-ocp-images.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp_far_edge/ztp-updating-gitops-ztp.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ztp-pulling-ocp-images_{context}"]
+= Enabling pulling ISO images for the desired {product-title} version
+
+To pull ISO images for the desired {product-title} version, update the `AgentServiceConfig` custom resource (CR) with references to the desired ISO and RootFS images that are hosted on the mirror registry HTTP server.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have {rh-rhacm} with MultiClusterHub enabled.
```

Comment:
PTAL at the repo guidelines for API object formatting: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting

You can either use a generic reference, like "... with multi-cluster hub enabled," or a specific reference, like "... with the `MultiClusterHub` object enabled." If you use a specific API reference, the camel case term must be enclosed in backticks and be followed by a noun.